scripts/aws/cfn/microservice.yml

```diff
@@ -119,6 +119,11 @@ Parameters:
     Type: String
     Description: 'enables check on physical address when creating notification'
     AllowedValues: [true, false]
+
+  IsMvpDefaultValue:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Parameter IsMvpDefaultValue is missing required property Type